packages/frontend/src/components/chart/Chart.tsx

```diff
@@ -120,10 +121,17 @@ export function Chart({
                 className="max-w-[135px] xs:max-w-none"
               />
             )}
-            {hasTvl && <CurrencyControls />}
+            {hasTvl && (
+              <div className="flex h-[2rem] items-end">
+                <CurrencyControls />
+                {hasDetailedTvl && <TokenControls tokens={tokens} />}
+              </div>
+            )}
             <ScaleControls />
           </div>
-          {hasTvl && <TokenControls tokens={tokens} />}
+          {hasTvl && !hasDetailedTvl && (
```

Comment:
This flag does not work on staging, the "Regular TVL" shows new modal

Reply:
On production it does not